### DISCUSSION TITLE
Idée de nom d'application mobile pour la visualisation de données?

### DISCUSSION THREAD

**Discussion by Thomass789** [drknzz/GitHub-Achievements#61] [Q&A]
Une idée s'il vous plait?

**Comment by youness240** [comment]


**Reply by Thomass789** [reply]
Merci !

**Reply by caaiopereira** [reply]
I am answering, Thanks!

**Comment by OwaisAzad** [comment]
Here are some suggestions for a name for a mobile data visualization app:

DataViz
VizFlow
Insightify
ChartScope
GraphiView
DataLens
AnalyzeNow
VisualizeMe
DataCape
StatTrack.

**Comment by CharlesCodebabel** [comment]
OmniscientData > OmDa
DonnéesOmniscientes > DoOm

**Comment by DanielR-spec** [comment]
Thank u for ur support :100:
